### ISSUE TITLE
Low acc with extracted features using v1.1

### ISSUE THREAD

**Issue by sherleens** [facebook/C3D#350]
Hi, I use v1.0 extract_image_features.bin to extract prob and get the same accuracy (39%) as the test.bin (40%). But I get much lower accuracy (11%) than the test.bin (39%) when I use extract_image_features.bin from v1.1. All the steps are same: same data - >same train/test split->training->testing->extracting features.  Do you any idea ?

**Comment by xuyunlu1030** [comment]
The same problem and wonder why! I guess the acc(40) from v1.0 is normal and it can be higher if using SVM following the features but the acc(11%) from v1.1 is quite confusing. Please let me know if you have any idea.

**Comment by dutran** [comment]
@sherleens Can you check if you turn of batch-norm to to use_global_statistics: true, v1.0 has no batch norm while 1.1 does
